holonote/annotate/display.py

```diff
@@ -336,8 +345,6 @@ def editable_enabled(self) -> bool:
     @editable_enabled.setter
     def editable_enabled(self, enabled: bool) -> None:
         self._editable_enabled = enabled
-        if not enabled:
-            self.clear_indicated_region()
```

Comment:
Was this also a leak?

Reply:
Yeah

Reply:
If you add it back, we can merge this PR

Reply:
Okay added